reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/Skip.kt

```diff
@@ -0,0 +1,33 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.completable.CompletableCallbacks
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+
+fun <T> Observable<T>.skip(count: Long): Observable<T> =
+    observable { emitter ->
+        val disposableWrapper = DisposableWrapper()
+        emitter.setDisposable(disposableWrapper)
```

Comment:
After switching to `observableUnsafe` just call `observer.onSubscribe` here.

Reply:
Fixed 👍